__tests__/app.test.js

```diff
@@ -0,0 +1,47 @@
+const app = require("../app.js");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const db = require("../db/connection.js");
+const request = require("supertest");
+
+beforeEach(() => seed(data));
+
+afterAll(() => db.end());
+describe("app", () => {
+  test("status 404 - invalid path", () => {
+    return request(app)
+      .get("/n")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("invalid path");
+      });
+  });
+  test("status 500 - internal server error", () => {}); // TODO how to test??
```

Comment:
is pretty difficult to test, but if you have it as last handle error with a console log you will always see what is happening and why if things goes wrong